[transitions] Allow to run Slide into a custom container (#26623)
mui-org/material-ui: docs/src/pages/components/transitions/transitions-es.md

{{"demo": "pages/components/transitions/SimpleSlide.js", "bg": true}}

### Slide from target DOM node

> The Slide component also receives `targetRef` prop, which is a reference to a DOM node. So the slide component would slide from the edge of the target DOM node.

Reviewer: ```suggestion
The Slide component also accepts `targetRef` prop, which is a reference to a DOM node. If this property is set, the slide component will slide from the edge of that DOM node.
```

> 
> ### Slide from target DOM node
> 
> The Slide component also accepts `targetRef` prop, which is a reference to a DOM node. If this property is set, the slide component will slide from the edge of that DOM node.
> 
> {{"demo": "pages/components/transitions/SimpleSlideFromTargetRef.js", "bg": true}}
> 
> 

Reviewer: ```suggestion
```

We are not manually changing the other languages
